prime-router/src/testIntegration/resources/settings/organizations.yml

```diff
@@ -575,4 +575,95 @@
       enrichmentSchemaNames:
         - "classpath:/metadata/fhir_transforms/common/suppress-non-npi.yml,classpath:/metadata/fhir_transforms/common/suppress-non-npi.yml"
       timeZone: null
-      dateTimeFormat: "OFFSET"
\ No newline at end of file
+      dateTimeFormat: "OFFSET"
+- name: "wa-phd"
```

Comment:
I'm guessing this was unintentional? We shouldn't be updating organizations.yml unless there is a need to add something for a smoke/integration test.

Reply:
This was not unintentional, this is the organizations.yml associated with the integration tests and I wanted their receiver settings to be taken into account in their integration test.